git_machete/cmd.py

```diff
@@ -3496,6 +3527,14 @@ def parse_options(in_args: List[str], short_opts: str = "", long_opts: List[str]
                 cli_ctxt.opt_no_edit_merge = True
             elif opt == "--no-interactive-rebase":
                 cli_ctxt.opt_no_interactive_rebase = True
+            elif opt == "--no-push":
+                cli_ctxt.opt_no_push = True
```

Comment:
Hmmm maybe to save one negation on the way... change the options to `opt_push` and `opt_push_untracked`, both defaulting to `True`? I wouldn't press on that for the existing options like `opt_no_detect_squash_merges` since they've got no "yes" version... but here it comes more naturally due to the existence o `--push`/`--push-untracked` flags 🤔